dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -288,10 +289,18 @@ def create_integration
         access_token_url: access_token_url,
         admin_email: admin_email
       )
-      render status: :ok, json: {body: I18n.t('lti.create_integration_success')}
-    else
-      render status: :conflict, json: {error: I18n.t('lti.error.integration_exists')}
+
+      @integration_status = :created
     end
+    render 'lti/v1/integration_status'
+  end
+
+  # GET /lti/v1/integration
+  # Displays the onboarding portal for creating a new LTI Integration
+  def new_integration
+    @form_data = {}
+    @form_data[:lms_platforms] = Policies::Lti::LMS_PLATFORMS.keys.map(&:to_s)
```

Comment:
I could move this logic to the `integration.html.haml` file, not sure if it's better to have this in the controller or view.